grammar.js

```diff
@@ -67,7 +71,7 @@ module.exports = grammar({
       $.import_from_statement,
       $.print_statement,
       $.assert_statement,
-      $.expression_statement,
+      $._expression_statement,
```

Comment:
@maxbrunsfeld After thinking on it some more, I don't think having `expression_statement` in the tree gives us any value. We currently don't show `simple_statement` or `compound_statement` in the tree. There were already a number of places where `expression_statement` was showing up in the tree for single `(identifier)` nodes prior to my changes, too. Curious what you think about this change, as it represents a pretty significant shift.